src/assets/Generator.Shared/MgmtPlaneOperationHelpers.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Concurrent;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core
+{
+    internal static class MgmtPlaneOperationHelpers
```

Comment:
For these classes that will end up in code lets stick with Arm as the prefix

Reply:
Changed